styles.css

```diff
@@ -123,11 +122,9 @@ a:first-child  {
 .count{
   color: orange;
 }
-/* View Overlay End*/
-
-
-/* Tablet */
+/* --------------View Overlay End--------------*/
 
```

Comment:
Great use of breakpoints! You do a nice job rearranging your cards as the size of the window grows and shrinks.

I think there's an opportunity to bring some of this logic into your `nav` styling- there are a few places where your `Naruto` text hops down below the `One Piece` link. I would consider adding a few more breakpoints in to help solve this- [here is documentation from Bootstrap](https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints) that outline a potential breakpoint schema based on popular screen sizes. Bootstrap is a styling toolkit that would be cool for you to explore as stretch tech in Mod 4! From there, I'd consider adding some grid/flexbox to the `nav` so that the cells holding your `a` tags move with the page.
![nav-breakpoint](https://user-images.githubusercontent.com/62263439/114324881-090ffe80-9aea-11eb-8b18-92ed739a96d6.gif)